src/main/java/io/kestra/plugin/shopify/products/Create.java

```diff
@@ -0,0 +1,245 @@
+package io.kestra.plugin.shopify.products;
+
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.io.IOException;
+import java.lang.InterruptedException;
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.property.Property;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.shopify.AbstractShopifyTask;
+import io.kestra.plugin.shopify.models.Product;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import java.net.URI;
+import java.util.Map;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Create a new product in Shopify store",
+    description = "Create a new product with the specified details."
+)
+@Plugin(
+    examples = {
+        @Example(
+        title = "Create a simple product",
+        full = true,
+        code = """
+        id: shopify_create_product
+        namespace: company.team
+
+        tasks:
+          - id: create_product
+            type: io.kestra.plugin.shopify.products.CreateProduct
+            storeDomain: my-store.myshopify.com
+            accessToken: "{{ secret('SHOPIFY_ACCESS_TOKEN') }}"
+            title: "New T-Shirt"
+            bodyHtml: "<p>A comfortable cotton t-shirt</p>"
+            vendor: "My Brand"
+            productType: "Clothing"
+            tags: "t-shirt,clothing,cotton"
+            status: "active"
+        """
+        )
+    }
+)
+public class CreateProduct extends AbstractShopifyTask implements RunnableTask<CreateProduct.Output> {
+
+    @Schema(
+        title = "Product title",
+        description = "The title of the product"
+    )
+    private Property<String> title;
+
+    @Schema(
+        title = "Product description HTML",
+        description = "The description of the product in HTML format"
+    )
+    private Property<String> bodyHtml;
+
+    @Schema(
+        title = "Vendor",
+        description = "The vendor of the product"
+    )
+    private Property<String> vendor;
+
+    @Schema(
+        title = "Product type",
+        description = "The product type"
+    )
+    private Property<String> productType;
+
+    @Schema(
+        title = "Tags",
+        description = "Comma-separated list of tags"
+    )
+    private Property<String> tags;
```

Comment:
It's more appropriated:

```suggestion
    private Property<List<String>> tags;
```